apps/src/studio/studio.js

```diff
@@ -1040,6 +1040,10 @@ Studio.onTick = function() {
     Studio.succeededTime = currentTime;
   }
 
+  if (!animationOnlyFrame) {
+    Studio.executeQueue('whenTouchGoal');
```

Comment:
more specifically, i see that we call handler for this event, and callHandler has this in the blockly codepath:

// Note: we skip executing the code if we have not completed executing
// the cmdQueue on this handler (checking for non-zero length)

does this mean that the 'whenTouchGoal' cmdQueue can be already have things in it and that's why it doesn't get called in blockly, or is there another reason callHandler isn't enough?